cli/src/commands/split.rs

```diff
@@ -120,6 +128,20 @@ don't make any changes, then the operation will be aborted.
         .write()?;
 
     // Create the second commit, which includes everything the user didn't select.
+    // TODO(emesterhazy): Maybe there's a prettier way to write this or we could
```

Comment:
I just tried this and it didn't work very well since the call to `description_template_for_commit` needs access to the base tree and tree for the second commit, which are different depending on whether we're splitting with --siblings or not. This is a pretty large block of code itself which has to be duplicated if those arguments aren't available in the outer scope.

I split the large let statement into two, which makes it a little more readable I suppose. In general this ternary pattern looks worse in Rust than in C++, but I'm not sure there's much we can do without a bigger refactor. We could move the code for creating the second commit to a function, but it would still by roughly the same number of lines of code.

Maybe it's ok as-is for now. If we make other changes to split (like allowing multiple splits the way mercurial does) then we can do further refactoring then. WDYT?